src/bin/pgcopydb/copydb.h

```diff
@@ -436,15 +436,17 @@ bool copydb_process_table_data_worker(CopyDataSpec *specs);
 bool copydb_process_table_data_with_workers(CopyDataSpec *specs);
 
 bool copydb_copy_table(CopyDataSpec *specs, PGSQL *src, PGSQL *dst,
-					   CopyTableDataSpec *tableSpecs);
+					   CopyTableDataSpec *tableSpecs,
+					   CopyProgressCallback onCopyProgress,
+					   void *onCopyProgressContext);
```

Comment:
Nitpicking: all the other callback signatures have the context first, the callback function next. What about doing the same here?